tests/testthat/test-compute_centiloid.R

```diff
@@ -0,0 +1,204 @@
+# ---- Valid combination tests ----
```

Comment:
Great job with these tests! We can even explore snapshot testing for the error/warnings, but it might be a bit overkill. Let's discuss next week and not let it stop merging this PR.